<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>Dialog</class>
    <widget class="QDialog" name="Dialog">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>260</width>
                <height>230</height>
            </rect>
        </property>
        <property name="minimumSize">
            <size>
                <width>260</width>
                <height>230</height>
            </size>
        </property>
        <property name="maximumSize">
            <size>
                <width>260</width>
                <height>230</height>
            </size>
        </property>
        <property name="windowTitle">
            <string>Login</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
            <item>
                <layout class="QFormLayout" name="formLayout">
                    <item row="0" column="0">
                        <widget class="QLabel" name="laUsername">
                            <property name="text">
                                <string>&amp;User</string>
                            </property>
                            <property name="buddy">
                                <cstring>ui_username</cstring>
                            </property>
                        </widget>
                    </item>
                    <item row="0" column="1">
                        <widget class="QLineEdit" name="ui_username">
                            <property name="maximumSize">
                                <size>
                                    <width>200</width>
                                    <height>16777215</height>
                                </size>
                            </property>
                        </widget>
                    </item>
                    <item row="1" column="0">
                        <widget class="QLabel" name="laPassword">
                            <property name="text">
                                <string>&amp;Password</string>
                            </property>
                            <property name="buddy">
                                <cstring>ui_password</cstring>
                            </property>
                        </widget>
                    </item>
                    <item row="1" column="1">
                        <widget class="QLineEdit" name="ui_password">
                            <property name="maximumSize">
                                <size>
                                    <width>200</width>
                                    <height>16777215</height>
                                </size>
                            </property>
                            <property name="echoMode">
                                <enum>QLineEdit::Password</enum>
                            </property>
                        </widget>
                    </item>
                    <item row="2" column="0">
                        <spacer name="frm_v_spacer">
                            <property name="orientation">
                                <enum>Qt::Vertical</enum>
                            </property>
                            <property name="sizeHint" stdset="0">
                                <size>
                                    <width>20</width>
                                    <height>24</height>
                                </size>
                            </property>
                        </spacer>
                    </item>
                    <item row="3" column="0">
                        <widget class="QLabel" name="laHostname">
                            <property name="text">
                                <string>&amp;Hostname</string>
                            </property>
                            <property name="buddy">
                                <cstring>ui_hostname</cstring>
                            </property>
                        </widget>
                    </item>
                    <item row="3" column="1">
                        <widget class="QComboBox" name="ui_hostname">
                            <property name="maximumSize">
                                <size>
                                    <width>200</width>
                                    <height>16777215</height>
                                </size>
                            </property>
                        </widget>
                    </item>
                    <item row="4" column="0">
                        <widget class="QLabel" name="laPort">
                            <property name="text">
                                <string>&amp;Port</string>
                            </property>
                            <property name="buddy">
                                <cstring>ui_port</cstring>
                            </property>
                        </widget>
                    </item>
                    <item row="4" column="1">
                        <widget class="QLineEdit" name="ui_port">
                            <property name="maximumSize">
                                <size>
                                    <width>80</width>
                                    <height>16777215</height>
                                </size>
                            </property>
                        </widget>
                    </item>
                </layout>
            </item>
            <item>
                <spacer name="verticalSpacer">
                    <property name="orientation">
                        <enum>Qt::Vertical</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                        <size>
                            <width>20</width>
                            <height>60</height>
                        </size>
                    </property>
                </spacer>
            </item>
            <item>
                <widget class="QLabel" name="lbl_status">
                    <property name="enabled">
                        <bool>true</bool>
                    </property>
                    <property name="sizePolicy">
                        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                            <horstretch>1</horstretch>
                            <verstretch>0</verstretch>
                        </sizepolicy>
                    </property>
                    <property name="text">
                        <string>Status</string>
                    </property>
                </widget>
            </item>
            <item>
                <layout class="QHBoxLayout" name="bottom_h_layout">
                    <item>
                        <widget class="QPushButton" name="btn_connect">
                            <property name="enabled">
                                <bool>false</bool>
                            </property>
                            <property name="text">
                                <string>Connect</string>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <widget class="QPushButton" name="pbCancel">
                            <property name="text">
                                <string>Cancel</string>
                            </property>
                        </widget>
                    </item>
                </layout>
            </item>
        </layout>
    </widget>
    <resources/>
    <connections>
        <connection>
            <sender>pbCancel</sender>
            <signal>clicked()</signal>
            <receiver>Dialog</receiver>
            <slot>reject()</slot>
            <hints>
                <hint type="sourcelabel">
                    <x>189</x>
                    <y>224</y>
                </hint>
                <hint type="destinationlabel">
                    <x>120</x>
                    <y>124</y>
                </hint>
            </hints>
        </connection>
    </connections>
</ui>
